packages/cli/src/commands/terragrunt/apply.ts

```diff
@@ -0,0 +1,63 @@
+import pc from "picocolors";
+import { printHelpInformation } from "../../util/print-help-information";
+import { progressMessage } from "../../util/progress-message";
+import type { BaseContext } from "clipanion";
+
+export function apply({
+  context,
+  verbose = false,
+  workingDirectory = ".",
+}: {
+  context: BaseContext;
+  verbose?: boolean;
+  workingDirectory?: string;
+}): 0 | 1 {
```

Comment:
It would be nicer to return structured objects rather than 0,1 / true,false. For example: {error: true/false, error_message: null / "something"}.

Reply:
What would the message be used for? I'm struggling to find the reasoning. The error message and/or `stderr` are printed for the user.